support.c

```diff
@@ -246,7 +269,7 @@ void set_source_ip(const char *tac_source_ip) {
 int _pam_parse_arg (const char *arg, char* current_secret, uint current_secret_buffer_size) {
     int ctrl = 0;
 
-    if (!strcmp (arg, "debug")) { /* all */
+    if (!strcmp (arg, "debug") || !strcmp (arg, "debug=on")) { /* all */

```

Comment:
> || !strcmp (arg, "debug=on") [](http://example.com/codeflow?start=30&length=29)

Is it really needed?

Reply:
Yes this is necessary for debug.
The reason of this change is because the tacplus_nss.conf config file using debug=on to enable debug. so we need change code here to enabled debug with the 'debug=on' setting.

Reply:
I am not aware of the syntax of `tacplus_nss.conf`. I found a sample [https://github.com/stemid/tacacs/blob/master/libnss-tacplus/tacplus_nss.conf](https://github.com/stemid/tacacs/blob/master/libnss-tacplus/tacplus_nss.conf). Will you support `debug=1` ?